src/kirby/components/icon/icon.component.tns.html

```diff
@@ -1 +1 @@
-<Label [text]="icon?.charCode" [style.fontFamily]="fontFamily"></Label>
+<Label [text]="icon?.charCode" [style.fontFamily]="fontFamily" [ngClass]="{'outline': outline}"></Label>
```

Comment:
Consider simplifying:
```suggestion
<Label [text]="icon?.charCode" [style.fontFamily]="fontFamily" [class.outline]="outline"></Label>
```